src/components/toastNotificationsList/toastNotificationsList.js

```diff
@@ -21,46 +25,44 @@ class ToastNotificationsList extends React.Component {
   };
 
   render() {
-    const { toasts, paused } = this.props;
+    const { toasts, timeout } = this.props;
 
     return (
-      <ToastNotificationList>
+      <AlertGroup isToast className="quipucords-toast-notifications__alert-group">
```

Comment:
required:
lets update these things
- add `isLiveRegion` to AlertGroup
- flip from `{toasts && toasts.map...` towards `{toasts?.map...`
- title attribute, remove the unnecessary nullish coalescing operator, replace it with an OR `||`... nullish coalescing has its time and place, but old skool truthy/falsy works just as well, especially in the scenario of someone passing an empty string
- lets be consistent with our ternary... replace `toast.header && toast.message ? ... : ''` with the format you used in the title attribute, `(toast.header && toast.message) || ''`